src/elements/hx-progress/index.spec.js

```diff
@@ -0,0 +1,61 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-progress> component tests
+ *
+ * @type HXProgressElement
+ *
+ */
+describe('<hx-progress> component tests', () => {
+    const template = '<hx-progress>';
+
+    describe('instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+    });
+
+    describe('test $onConnect method', () => {
```

Comment:
@lalithkarikelli good work here.  👍 

Also, please add a test to check for default attribute `role`.  And another test to verify that the value of `role` is `progressbar`.  Please see example below.